src/forms/input/index.tsx

```diff
@@ -47,7 +47,17 @@ export class Input extends React.PureComponent<InputProps> {
     }
 
     render() {
-        const {className, disabled, helperText, label, isBoxed, placeholder, size, type} = this.props;
+        const {
+            className, //prettier
+            disabled,
+            helperText,
+            label,
+            isBoxed,
+            placeholder,
+            size,
+            type,
+            children,
```

Comment:
I think we also need to add  prop named as 'children' in InputProps.
For example ,children?: ReactNode | ReactNode[];

Reply:
@akate1, this is not required as `children` is a property always available in React components.

Reply:
ok got it.